pkg/machines/manifest.json.in

```diff
@@ -9,5 +9,6 @@
         "path": "index.html",
         "order": 60
      }
-  }
+  },
+  "content-security-policy": "default-src 'self' 'unsafe-inline' data:"
```

Comment:
Ah, having it in the commit description (not just the PR) seems okay. Unfortunately JSON files can't have comments.